ssl/ssl_ciph.c

```
@@ -498,12 +499,12 @@ static void do_load_builtin_compressions(void)
         }
     }
     CRYPTO_mem_ctrl(CRYPTO_MEM_CHECK_ENABLE);
+    return 1;
```

And I guess the OPENSSL_malloc() failure should instead return an error